csrc/fusion.h

```diff
@@ -451,13 +455,14 @@ class NVF_API Fusion : public IrContainer {
 
   //! Clear Expr's from TV uses that are not required to produce outputs from
   //! inputs. Only other place this is used (other than Fusion) is in
-  //! Val::uses()
-  void resetTvUses();
+  //! Val::uses(). Also populate the all_tvs_ entry with all the used TVs.
+  void resetAllTvsAndUses();
 
   //! Declare that TensorView uses need to be updated (but don't actually do
   //! the update).
   void invalidateTvUses() {
-    all_tv_uses_valid_ = false;
+    all_tvs_and_uses_valid_ = false;
```

Comment:
This is an issue with the existing code, but this doesn't seem to be consistently used. We should just either remove it or rename it to something `invalidateAllTvsAndUses` and use it consistently.